src/switch/switch.stories.ts

```diff
@@ -10,7 +10,10 @@ storiesOf("Switch", module).addDecorator(
 )
 	.addDecorator(withKnobs)
 	.add("Basic", () => ({
-		template: `<ibm-switch [disabled]="disabled"></ibm-switch>`,
+		template: `
+		<ibm-switch [disabled]="disabled"></ibm-switch>
```

Comment:
Indent template code by a tab.
```suggestion
			<ibm-switch [disabled]="disabled"></ibm-switch>
```